libc/test/src/__support/arg_list_test.cpp

```diff
@@ -120,7 +120,7 @@ TEST(LlvmLibcArgListTest, TestStructTypes) {
 }
 
 // Test vector extensions from clang.
-#if __has_attribute(ext_vector_type)
+#if !defined(LIBC_TARGET_OS_IS_WINDOWS) && __has_attribute(ext_vector_type)
```

Comment:
Should probably explicitly `#include "libc/src/__support/macros/properties/os.h"` in files where use of `LIBC_TARGET_OS_IS_WINDOWS` is being added.

Reply:
Updated.